.gitignore

```diff
@@ -1,2 +1 @@
-vendor/
-.bundle/
```

Comment:
these are used to ignore bundler for running the spec/test suite. Is there a specific reason you're removing them?

Reply:
@hone I thought these were unused, since I don't ever remember bundler installing things to the current working directory, only the global locations - but perhaps that differs across bundler versions or when using the commit gems to repo workflow etc.

I'll restore them for now.